src/mocks/handlers.js

```diff
@@ -2,9 +2,9 @@ import { http, HttpResponse } from "msw";
 import { responses } from "./responses";
 import { setOidcUser } from "./login";
 
-const CLIENT_URL = process.env["REACT_APP_CLIENT_URL"];
-const OIDC_AUTHORITY_URL = process.env["REACT_APP_OIDC_AUTHORITY_URL"];
-const OIDC_CONFIG_URL = OIDC_AUTHORITY_URL + ".well-known/openid-configuration";
+const CLIENT_URL = new URL(String(process.env["REACT_APP_CLIENT_URL"]));
+const OIDC_AUTHORITY_URL = new URL(String(process.env["REACT_APP_OIDC_AUTHORITY_URL"]), CLIENT_URL);
+const OIDC_CONFIG_URL = new URL(".well-known/openid-configuration", OIDC_AUTHORITY_URL)
```

Comment:
If you do this, then the URL constructor will do `https://data.ghga-dev.de/.well-known`, since `/.well-known` specifies the URL should start at the absolute base of the path, rather than take the base root of the OIDC Authority URL. I don't think that's how it was programmed to be before.

Reply:
Yes, but it is the correct way to do it. The code before assumed that the OIDC Authority URL does not contain a path, which should always be the case, so it worked as well.

Reply:
Done